packages/build/webpack/lib.template.xml

```diff
@@ -7,12 +7,9 @@
             // This could theoretically be done by poking the "compilation" object from webpack but I could not
             // find a strategy that works.
             [...files.css, ...files.js].forEach((filePath) => {
-                if (filePath.indexOf('/vendors~') > -1 ||
-                    filePath.indexOf('/' + options.name + '~') > -1 ||
-                    filePath.indexOf('/' + options.name + '.css') > -1 ||
-                    filePath.indexOf('/' + options.name + '.js') > -1) {
+                if (filePath.indexOf('/vendors~') > -1 || filePath.indexOf(options.name) > -1) {
```

Comment:
note that the vendors check is different. sorry missed it the first check

Reply:
Derp. Fixed.